docs/configuration/v2.x/runtime.md

```diff
@@ -66,9 +66,36 @@ by providing the metric information to the configured sinks.
 
 As of today, we support the follow sinks:
 
+- **Atlassian Statuspage**
 - **Prometheus Scraping Endpoint**
 - **StatsD**
 
+### Atlassian Statuspage
+
+![Availability Badge](https://img.shields.io/badge/Available%20Starting-v2.0-green.svg)
+
+In order to expose a Prometheus Scraping endpoint, you'll need to configure the sink:
+
+- `atlassianStatuspage.pageId` - Defines the id of the Atlassian Statuspage to report to.
+- `atlassianStatuspage.systemMetricMapping` - Defines a mapping of the scraped metric by Promitor and to which
+ Atlassian Statuspage system metric it should be reported to. Here's what we expect:
+  - `id` - Id of the Atlassian Statuspage system metric
+  - `promitorMetricName` - Name of the Promitor metric which needs to be reported
+
+Next to that, `PROMITOR_ATLASSIAN_STATUSPAGE_APIKEY` environment variable is required which contains the API Key
+ for Atlassian Statuspage.
+
+```yaml
+metricSinks:
+  atlassianStatuspage:
+    pageId: XXX          # Mandatory
+    systemMetricMapping: # Mandatory to have at least one mapping
+    - id: ABC
+      promitorMetricName: promitor_demo_appplan_percentage_cpu
+```
+
+> :warning: Metric labels are not supported.
```

Comment:
It is indeed only aimed at reporting a single resource so that business KPIs in Application Insights or general performance once such as request rate, etc. can reported.

I was going to improve the docs but your recommendation is to add a validation rule for:
- Multiple configured resources
- Optionally block resource discovery?

Reply:
That makes sense. I think if it only really works with one resource, or you could get confusing behaviour if you used multiple resources, it makes sense to me to add validation to either warn or block someone from linking a metric with multiple resources to status page.

I guess if you started by blocking then it could always be changed later if someone came along with a use-case that needed it.

Reply:
That's what I was thinking as well!